imqsauth.go

```diff
@@ -314,12 +314,17 @@ func permShow(icentral *auth.ImqsCentral, identityColumnWidth int, identity stri
 	groupCache := map[authaus.GroupIDU32]string{}
 	if perm, e := icentral.Central.GetPermit(user.UserId); e == nil {
 		if groups, eDecode := authaus.DecodePermit(perm.Roles); eDecode == nil {
-			if groupNames, eGetNames := authaus.GroupIDsToNames(groups, icentral.Central.GetRoleGroupDB(), groupCache); eGetNames == nil {
+			groupNames, eGetNames := authaus.GroupIDsToNames(groups, icentral.Central.GetRoleGroupDB(), groupCache)
+			if eGetNames != nil && groupNames == nil {
+				permStr = fmt.Sprintf("Error converting group IDs to names: %v\n", eGetNames)
+				icentral.Central.Log.Errorf("permShow: Error converting group IDs to names, returning (%v)", eGetNames)
```

Comment:
Base on the rest of the code in this function. We should not be writing the error here, as setting `permStr` is enough.